### PR TITLE
[release/v8.6] Update software tag

### PR DESCRIPTION
**Pull request by chetan-zilliqa**
## Backward Compatibility

- [ ] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [ ] **ready for review**

### Integration Test (Core Team)

- [ ] local machine test

- [ ] small-scale cloud test

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -4,6 +4,8 @@
 6
 === [Zilliqa]Fix Version ===
 0
+=== [Zilliqa]rc version ===
+0
```
Comment (JamesHinshelwood) [inline VERSION:8]: @yaron-zilliqa Can you take a look at this one? Do we need to update the stuff in `docker/Makefile` to respect this?

Comment (yaron-zilliqa) [reply VERSION:8]: Don't recall these were ever respected. But basically it'd be good to, I suppose.

Comment (yaron-zilliqa) [reply VERSION:8]: Anyway, I think for now it can be ignored or removed.

Comment (JamesHinshelwood) [reply VERSION:8]: Looks like they are now: https://github.com/Zilliqa/Zilliqa/blob/4e74fea415feac83a926a0d87cee22cea6c57c1c/docker/Makefile#L9-L11
(I can't remember if I changed this or not)

**Pull request merged**